### PR TITLE
FLUID-6153: Updating infusion to address scroll bar issue

### PR DESCRIPTION
**Pull request by jobara**
https://issues.fluidproject.org/browse/FLUID-6153

### CODE REVIEW

File: lib/infusion/VERSION.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-The version of Infusion included in this folder was created using a custom build from commit [f8d92a56023a3ae6c1c2a20773f7ac44e1514d31](https://github.com/fluid-project/infusion/tree/f8d92a56023a3ae6c1c2a20773f7ac44e1514d31) of the Infusion master branch [https://github.com/fluid-project/infusion/](https://github.com/fluid-project/infusion/) using the command line:
+The version of Infusion included in this folder was created using a custom build from commit [56cc1caafba6907f1ecf93cc3baf17b0e357ee97](https://github.com/fluid-project/infusion/tree/56cc1caafba6907f1ecf93cc3baf17b0e357ee97) of the Infusion master branch [https://github.com/fluid-project/infusion/](https://github.com/fluid-project/infusion/) using the command line:
```
Comment (cindyli) [inline lib/infusion/VERSION.md]: Can you update the list after line 6 with the files being removed but were not on there, such as `infusion/src/components/textToSpeech/js/MockTTS.js`.

Comment (jobara) [inline lib/infusion/VERSION.md]: The entire textToSpeech directory is marked as to be removed already. So that file shouldn't have been included before.

Comment (cindyli) [inline lib/infusion/VERSION.md]: Right. Missed that.

**Comment by cindyli** [comment]
Merged at 9de3981233cbd1f878152e98bbc3f7ae6eaea3ed

**Pull request merged**